### PR TITLE
Prepare dtci

### PR DESCRIPTION
**Pull request by dzvancuks**
- Prepared handlers to create and parse Extended Common Flags and Extended Common Flags II.
- Fixed HasUASI function name

See https://www.etsi.org/deliver/etsi_ts/129000_129099/129060/13.06.00_60/ts_129060v130600p.pdf

After review I'll change target to upstream

### CODE REVIEW

File: gtpv1/ie/extended-common-flags.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+// found in the LICENSE file.
+
+package ie
+
+import "io"
+
+// NewExtendedCommonFlags creates a new ExtendedCommonFlags IE.
+//
+// Note: each flag should be set in 1 or 0.
+func NewExtendedCommonFlags(uasi, bdwi, pcri, vb, retloc, cpsr, ccrsi, unauthenticatedIMSI int) *IE {
```
Comment (ptsgr) [inline gtpv1/ie/extended-common-flags.go:12]: May be params should be `uint8`?

File: gtpv1/ie/extended-common-flags-ii.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+// found in the LICENSE file.
+
+package ie
+
+import "io"
+
+// NewExtendedCommonFlagsII creates a new ExtendedCommonFlagsII IE.
+//
+// Note: each flag should be set in 1 or 0.
+func NewExtendedCommonFlagsII(pmtsmi, dtci, pnsi int) *IE {
```
Comment (ptsgr) [inline gtpv1/ie/extended-common-flags-ii.go:12]: May be params should be `uint8`? (Just optimization)

**Review by ptsgr — APPROVED** [review]
LGTM

File: gtpv1/ie/extended-common-flags.go (hunk 1 above)
Comment (dzvancuks) [reply gtpv1/ie/extended-common-flags.go:12]: Made int to be consistent with `NewCommonFlags` function that uses int

**Review by valdeits — APPROVED** [review]

**Review by AlexanderKudzevich — APPROVED** [review]
Do we need to update UT here - /gtpv1/message/create-pdp-context-res_test.go?

**Comment by dzvancuks** [comment]
> Do we need to update UT here - /gtpv1/message/create-pdp-context-res_test.go?

It is a good idea but not really required. Tests in this lib are more like smoke tests. Not every IE is tested.

**Pull request merged**